job-descriptions/software-engineer.md

```diff
@@ -34,69 +30,21 @@ As a senior member of the team, you will:
-- Implement auto-complete typeahead for search queries.
-- Write robust end-to-end tests for our website and browser extensions.
-- Create the Sourcegraph browser extension for Chrome, Firefox, and Safari.
-
-##### Clients
-
-- Create a `src` CLI command that makes it easy for customers to perform common operations against our APIs.
-- Create Sourcegraph editor extensions (e.g. Visual Studio Code, Atom, Intellij, Sublime) that allow developers to use Sourcegraph without breaking their flow.
-
-##### Backend
-
-- Create a system to perform fast, real-time, cross-repository `git grep`.
-- Create a platform that allows our customers to extend the functionality of Sourcegraph.
-- Make our code search and code intelligence scale to 10,000+ repositories and massive mono-repositories without overloading our customer's code hosts.
-- Add support for new code hosts to Sourcegraph.
-- Reduce P95 latency of requests to our API (e.g. hover tooltips, find references, go-to-definition).
-
-##### Operations
-
-- Streamline how customers configure and deploy Sourcegraph to a Kubernetes cluster.
-- Improve our CI infrastructure to be faster and more reliable.
-- Scale our Kubernetes cluster to meet traffic demands and avoid OOMs, while minimizing/optimizing the infrastructure costs for ourselves and for our customers.
-- Implement zero downtime deploys of Sourcegraph.
-- Standardize how we do logging and telemetry across all our systems.
-- Increase the observability of our systems so we can figure out what is wrong when issues are reported.
-
 ### Interview process
 
 1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU).
-
-    - Please explain why you are interested in Sourcegraph.
-    - Try out our product (e.g. code search and browser extension) and let us know what feedback you have.
-    - If you have any non-trivial code samples that you can share with us (e.g. your own open source projects, or contributions to other projects) we would love to look at that.
-
-2.  We schedule a 30 minute introductory phone call to tell you more about Sourcegraph, to share what we are looking for in a candidate and to find out what you are looking for in your next role.
-    
-    Let us know if you are interested in doing a paid project with us. This gives you a chance to see the kind of real world problems that we work on and gives us another data point to consider in case interviews aren't your forte.
-
+2.  We schedule a 30 minute introductory phone call to tell you more about Sourcegraph, to share what we are looking for in a candidate, and to find out what you are looking for in your next role.
 3.  We schedule a 1 hour technical phone interview.
-    - We will write some code in a tool like https://coderpad.io.
-    - We avoid trick questions and problems that require a single epiphany to solve.
 4.  We fly you out to our San Francisco office for a few more hours of interviews and to meet the team.

```

Comment:
We should clarify that we are happy to conduct all interviews remotely if that is the candidate's preference.

```suggestion
4.  We schedule a few more hours of technical and non-technical interviews. We are happy to fly you out to our San Francisco office to meet the team in person, or we can can also conduct the remaining interviews remotely, whatever works best for you.
```